src/main/java/org/jenkinsci/plugins/badge/ImageResolver.java

```diff
@@ -20,189 +20,219 @@
 * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 * THE SOFTWARE.
-*/
+ */
 package org.jenkinsci.plugins.badge;
 
 import hudson.model.BallColor;
-
-import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
 import java.util.HashMap;
 import java.net.MalformedURLException;
 import java.net.URL;
 
 import jenkins.model.Jenkins;
-
-import org.apache.commons.io.FileUtils;
 import org.apache.commons.io.IOUtils;
-
+/**
+ *
+ * @author dkersch
+ */
 public class ImageResolver {
-
+    /**
+     *
+     */
     private final HashMap<String, StatusImage[]> styles;
+    /**
+     *
+     */
     private final StatusImage[] defaultStyle;
-    public static final String COVERAGE_BASE_URL = "https://img.shields.io/badge/coverage";
+    /**
```

Comment:
[Checkstyle] WARNING: First sentence of Javadoc is incomplete (period is missing) or not present.